cobigen/cobigen-core/src/test/resources/testdata/unittest/config/properties/invalidConfigProperties/config.properties

```diff
@@ -0,0 +1,2 @@
+template-sets.installed=com.com.com:com.com:com.com:com;blabla:blabla:
+template-sets.hide=com.com.com:com.com:com.com:com;blabla:blabla:
```

Comment:
A more practical example would be nice here.

Reply:
What do you mean? This test is for using wrong coordinates that arent compliant with the maven naming syntax.